src/components/LineChart/LineChart.jsx

```diff
@@ -371,12 +380,14 @@ const LineChart = createClass({
 			yAxisTitleColor,
 			yAxisMin,
 			yAxisTooltipFormatter,
+			yAxisTooltipDataFormatter,
```

Comment:
Actually, this isn't the place to set default values for props. Seems like setting `yAxisTooltipFormatter = yAxisFormatter` would just overwrite the value passed in for `yAxisTooltipFormatter`. Am I missing something?